.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -1272,3 +1272,6 @@ stages:
       TAGS_TO_SKIP: "os=windows"
     jobs:
       - template: ./templates/e2e-template.yaml
+        parameters:
+          name: All Linux
```

Comment:
not necessary all linux tests, only for VHD build in the pipeline. User picks the VHDs.

Reply:
I think it'll still run all the tests tho - AFAIK there isn't a way to reduce the tests to just the built VHDs.

Reply:
It uses buildID env variable.

And run tests only for VHDs tagged with the build id.